app/src/main/java/org/dhis2/AppComponent.java

```diff
@@ -21,6 +21,7 @@
 import org.dhis2.data.service.workManager.WorkManagerModule;
```

Comment:
Using service locator solution in 2.9.1 this change is not necessary.

We should not add specific sms code in other flavors